server-worker/src/main/java/com/aliyun/emr/rss/service/deploy/worker/storage/PartitionFilesSorter.java

```diff
@@ -567,6 +579,7 @@ public void sort() {
         }
 
         ((DirectBuffer) paddingBuf).cleaner().clean();
+        memoryTracker.releaseSortMemory(reserveMemory);
```

Comment:
I think we should use initialReserveSingleSortMemory here because we reserved initialReserveSingleSortMemory bytes. And we'd better put this line inside finally.

Reply:
If old buffer size is not enough, it will call `expandBufferAndUpdateMemoryTracker()` and release the `initialReserveSingleSortMemory` when the first time re-allocating. And if buffer size is enough, the `reserveMemory` would be same with `initialReserveSingleSortMemory`. If we replace `reserveMemory` with `initialReserveSingleSortMemory` here, the memoryTracker would update wrong value which is smaller than the actual value.
```
private ByteBuffer expandBufferAndUpdateMemoryTracker(int oldCapacity, int newCapacity) {
      memoryTracker.releaseSortMemory(oldCapacity);
      memoryTracker.reserveSortMemory(newCapacity);
      return ByteBuffer.allocateDirect(newCapacity);
}
```